maker/src/position.rs

```diff
@@ -0,0 +1,220 @@
+use async_trait::async_trait;
```

Comment:
Honest rust question: why is this not in `./position/mod.rs`?

Reply:
It's optional to use `mod.rs` since Rust 2018, although I prefer not using it: https://doc.rust-lang.org/edition-guide/rust-2018/path-changes.html#no-more-modrs.

At some point @holzeis wanted to try out introducing `mod.rs`, but it's a bit annoying when you have multiple files open with the same `mod.rs` name. I would be keen to align it and remove all `mod.rs` files in favour of the newer standard.